Sources/GRPCCodeGen/Internal/Translator/ClientCodeTranslator.swift

```diff
@@ -22,6 +22,7 @@
 /// a representation for the following generated code:
 ///
 /// ```swift
+/// @available(macOS 13.0, *)
```

Comment:
These need to be for other platforms too: `@available(macOS 13.0, iOS 16.0, watchOS 9.0, tvOS 16.0, *)`